packages/react-core/src/components/EmptyState/EmptyStateHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
+import { EmptyStateIconProps } from './EmptyStateIcon';
+
+export interface EmptyStateHeaderProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the empty state header */
+  children?: React.ReactNode;
```

Comment:
Not a blocker, but wondering if we should try to differentiate `titleText` from `children` a little more. Maybe the description could be, "Content rendered inside the empty state header, either in addition to or instead of the titleText prop."

Reply:
Great idea! You are right that the documentation was a bit confusing. I updated it and added few comments to titleText and headingLevel props too.